pkg/proc/variables.go

```diff
@@ -776,7 +776,7 @@ func newVariable(name string, addr uint64, dwarfType godwarf.Type, bi *BinaryInf
 
 var constantMaxInt64 = constant.MakeInt64(1<<63 - 1)
 
-func newConstant(val constant.Value, mem MemoryReadWriter) *Variable {
+func newConstant(val constant.Value, bi *BinaryInfo, mem MemoryReadWriter) *Variable {
```

Comment:
Unless I'm missing something obvious, this new parameter doesn't seem to be used. I think it's meant to be used during the initialization of `v` right below (and then subsequently passed along as `v.bi` below).

Reply:
lol, fixed